root/search/components/EventResults.js

```diff
@@ -28,8 +28,9 @@ function buildResult(result, index) {
   return (
     <tr className={loopParity(index)} data-score={score} key={event.id}>
       <td>
-        <DescriptiveLink entity={event} />
+        <a href={'/event/'+event.id}>{event.name}</a>
```

Comment:
Disambiguation comment and cancellation are missing here, whereas only date should be removed.

Another way to do it is to add options `DescriptiveLink` and `EntityLink` to skip adding date for event.

Reply:
I agree with adding a prop like `showEventDate` to `EntityLink`. Not sure there's a reason to be using `DescriptiveLink` here since it's just a wrapper around `EntityLink` that doesn't appear to add anything for events.

Reply:
Sure, that seems like a far cleaner approach.

Reply:
@mwiencek why would we want to use showEventDate? Surely that shows the event date?

Reply:
Well, as @yvanzo pointed out we still want to show the disambiguation comment and "canceled" status, so you'd need:

```JSX
<EntityLink entity={event} showDisambiguation />
```

But `showDisambiguation` also shows the dates. So by adding a `showEventDate` flag (defaulting to true) you can set it to false there:

```JSX
<EntityLink entity={event} showDisambiguation showEventDate={false} />
```

Reply:
Ah right, I think I understand. I'd wrongly assumed that you could only add flags like this to add to a component. and hadn't thought about setting anything to false 😄 

Will commit this in a sec